tmail-backend/integration-tests/jmap/memory-jmap-integration-tests/src/test/java/com/linagora/tmail/james/MemoryFirebaseSubscriptionGetMethodTest.java

```diff
@@ -27,6 +29,7 @@ public class MemoryFirebaseSubscriptionGetMethodTest implements FirebaseSubscrip
             .build())
         .server(configuration -> MemoryServer.createServer(configuration)
             .overrideWith(new LinagoraTestJMAPServerModule())
-            .overrideWith(new FirebaseSubscriptionProbeModule()))
+            .overrideWith(new FirebaseSubscriptionProbeModule())
+            .overrideWith(binder -> binder.bind(FirebasePushClient.class).toInstance(FirebaseSubscriptionSetMethodContract$.MODULE$.firebasePushClient())))
```

Comment:
I just used `FirebaseSubscriptionSetMethodContract.firebasePushClient()` and it works.